src/app/shared/top-panel/user-panel/notifications-menu/notifications-menu.component.html

```diff
@@ -56,12 +56,12 @@
           </span>
         </span>
         <span class="notification-content read text-notice cursor-pointer"
```

Comment:
You can remove `read` and `text-notice` class from here, because they are added conditionally.

Reply:
Good catch!